dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -171,6 +175,58 @@ const buttonExpanded = css`
 	}
 `;
 
+const badgeDiameter = 20;
+const notificationColor = palette.error[400];
+
+const notificationBadgeStyles = css`
+	background-color: ${notificationColor};
+	color: ${palette.neutral[100]};
+	position: absolute;
+	top: 0;
+	left: 0;
+	min-width: ${badgeDiameter}px;
+	height: ${badgeDiameter}px;
+	border-radius: ${badgeDiameter / 2}px;
+	text-align: center;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 0 5px;
+	margin-left: -10px;
+	margin-top: -3px;
+`;
```

Comment:
Is there a reason to use absolute positioning and negative margins?

```css
  top: -3px;
  left: -10px;
```

If you’re using `display: flex;`, do you also need to `text-align: center;`?

Reply:
> If you’re using display: flex;, do you also need to text-align: center;?

Oh! I am using `text-align: center;`, or am I misunderstanding the question?!

Reply:
Not sure what the benefit of aligning both the text and element to centre…